### DISCUSSION TITLE
legacy-swc-helpers critical vulnerability

### DISCUSSION THREAD

**Discussion by juanfiallos07** [swc-project/swc#9581] [General]
Checking the [https://github.com/advisories/GHSA-9p8c-m465-cvpr](url)

If in my code I've only have legacy-swc-helpers  for alias, 

> "@swc/helpers@^0.4.14":
>   version "0.4.36"
>   resolved "https://registry.yarnpkg.com/@swc/helpers/-/helpers-0.4.36.tgz#fcfff76ed52c214f357e8e9d3f37b568908072d9"
>   integrity sha512-5lxnyLEYFskErRPenYItLRSge5DjrJngYKdVjRSrWfza9G6KkgHEXi0vUZiyUeMU5JfXH1YnvXZzSp8ul88o2Q==
>   dependencies:
>     legacy-swc-helpers "npm:@swc/helpers@=0.4.14"
>     tslib "^2.4.0"

    
    

> "legacy-swc-helpers@npm:@swc/helpers@=0.4.14":
>   version "0.4.14"
>   resolved "https://registry.yarnpkg.com/@swc/helpers/-/helpers-0.4.14.tgz#1352ac6d95e3617ccb7c1498ff019654f1e12a74"
>   integrity sha512-4C7nX/dvpzB7za4Ql9K81xK3HPxCpHMgwTZVyf+9JQ6VUbn9jjZVN7/Nkdz/Ugzs2CSjqnL/UPXroiVBVHUWUw==
>   dependencies:
>     tslib "^2.4.0"

is there any issue I should be concerned about?

**Comment by kdy1** [comment]
Nope, see https://github.com/swc-project/swc/discussions/9400
